src/components/Topics/List/ListItem.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import cx from 'classnames';
+import { NavLink } from 'react-router-dom';
+import { TopicWithDetailedInfo } from 'lib/interfaces';
+
+const ListItem: React.FC<TopicWithDetailedInfo> = ({
+  name,
+  internal,
+  partitions,
+}) => {
+  const outOfSyncReplicas = React.useMemo(() => {
+    if (partitions === undefined || partitions.length === 0) {
+      return 0;
+    }
+
+    return partitions.reduce((memo: number, { replicas }) => {
+      const outOfSync = replicas.filter(({ inSync }) => !inSync)
+      return memo + outOfSync.length;
+    }, 0);
+  }, [partitions])
+
+  return (
+    <tr>
+      <td>
+        <NavLink exact to={`topics/${name}`} activeClassName="is-active" className="title is-6">
+          {name}
+        </NavLink>
+      </td>
+      <td>{partitions.length}</td>
+      <td>{outOfSyncReplicas}</td>
+      <td>
+        <div className={cx('tag is-small', internal ? 'is-light' : 'is-success')}>
+          {internal ? 'Internal' : 'External'}
```

Comment:
What about move that values to constants or kinda of translations service?

Reply:
Oh. I want to avoid using any translation services